apps/search/style/search.css

```diff
@@ -43,38 +43,27 @@ body, a {
   display: flex;
 }
 
-section.suggestions {
-  margin: 0 0.7rem;
-  font-size: 1.4rem;
-  font-style: italic;
-  overflow: hidden;
-  height: 3.5rem;
+#suggestions {
+  font-size: 1.6rem;
+  overflow-x: hidden;
+  margin-right: 1.6rem;
 }
 
-section.suggestions ul {
+#suggestions ul {
   list-style: none;
-  color: #00AACC;
   white-space: nowrap;
-  overflow-x: scroll;
   padding-left: 0;
-  height: 5rem;
 }
 
-section.suggestions ul li {
-  display: inline-block;
-  margin-left: 1rem;
-  padding: 0.3rem 0.7rem 0.9rem 0;
+#suggestions li {
+  height: 6rem;
+  line-height: 6rem;
   margin: 0;
-}
-
-section.suggestions ul li+li:before {
-  content: "\2022";
-  padding-right: 0.7rem;
-  color: #666;
-}
-
-section.suggestions ul li:first-child {
-  padding-left: 0.7rem;
+  border-bottom: 0.1rem solid rgba(255, 255, 255, 0.05);
```

Comment:
nit: The border at the bottom currently lacks padding on the left which doesn't quite meet the visual spec.